### PR TITLE
0041-transaction-pool-size.md

### PR DESCRIPTION
**Pull request by bijanshahrokhi**
Thank you for contributing to Coda! Please see `CONTRIBUTING.md` if you haven't
yet. In that doc, there are more details around how to start our CI.

Explain your changes here.

Explain how you tested your changes here.

Checklist:

- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

Closes #0000
Closes #0000

### CODE REVIEW

**Comment by psteckler** [comment]
See #6562, where I recorded the results of experiments yielding txn pool memory requirements.

It looks like 1kb per transaction is a conservative estimate. The 1.5kb number I gave earlier was with a smaller pool.

File: rfcs/transaction-pool-size [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Given there is no evidence of block producers demanding more than 5 hours of transactions in their queue and Mina is not prioritizing high TPS at this time, it is recommended to stick to the 10MB transaction pool size. This translates to about 8 hours of queue.  
+
+Zaki (Comos) & Luke (Tezos) also think that 8 hours of queue time is sufficient for us as a start. 
+
+## Mina Protocol mainnet 
+
+TPS: 0.2 
+TX size: 1.5KB 
+Transaction Pool Size: 10MB 
+Queue in hours: 10MB / (0.2 TPS x 1.5KB) = 30K seconds ~ 8 hours
```
Comment (psteckler) [inline rfcs/transaction-pool-size:34]: See #6562.

Comment (bijanshahrokhi) [reply rfcs/transaction-pool-size]: That's great. I updated the numbers to reflect this. We are at an even better position.

**Comment by mrmr1993** [comment]
Old and stale. Closing.

**Pull request closed without merging**